initfiles/componentfiles/configxml/esp.xsd.in

```diff
@@ -163,12 +163,21 @@
                                             </xs:appinfo>
                                         </xs:annotation>
                                     </xs:attribute>
+                                    <xs:attribute name="path" type="xs:string" use="optional" default="/">
+                                        <xs:annotation>
+                                            <xs:appinfo>
+                                                <tooltip>The logical path of a resource used for authentication.</tooltip>
```

Comment:
these paths are use for Authorization, correct?

Reply:
As far as I know.